base/setup/reactos/lang/tr-TR.rc

```diff
@@ -203,88 +204,88 @@ END
-Spaces are not allowed."
+    IDS_ERROR_INVALID_INSTALLDIR_CHAR_TITLE "Geçersiz karakter"
+    IDS_ERROR_INVALID_INSTALLDIR_CHAR "Geçerli karakterler şunlardır:\n\
+alfanümerik (a-z, A-Z, 0-9), ve\n . \\ - _\n\
+Boşluklara izin verilmez."
 
-    IDS_ERROR_DIRECTORY_NAME_TITLE "Invalid installation path"
-    IDS_ERROR_DIRECTORY_NAME "The ReactOS installation path must follow the DOS 8.3 naming scheme, \
-and only contain letters, digits, dashes and periods. Spaces are not allowed."
+    IDS_ERROR_DIRECTORY_NAME_TITLE "Geçersiz kurulum yolu"
+    IDS_ERROR_DIRECTORY_NAME "ReactOS kurulum yolu DOS 8.3 adlandırma şemasını takip etmelidir, 
+\n ve yalnızca harf, rakam, tire ve nokta içerir. Boşluklara izin verilmez."
 
-    IDS_ERROR_CREATE_PARTITION_TITLE "Create partition"
-    IDS_ERROR_CREATE_PARTITION "Failed to create a new partition."
+    IDS_ERROR_CREATE_PARTITION_TITLE "Bölüm oluştur"
+    IDS_ERROR_CREATE_PARTITION "Yeni bir bölüm oluşturulamadı."
 
-    IDS_WARN_DELETE_PARTITION_TITLE "Delete partition?"
-    IDS_WARN_DELETE_PARTITION "Are you sure you want to delete the selected partition?"
-    IDS_WARN_DELETE_MBR_EXTENDED_PARTITION "Are you sure you want to delete the selected extended partition and ALL the logical partitions it contains?"
+    IDS_WARN_DELETE_PARTITION_TITLE "Bölüm sil?"
+    IDS_WARN_DELETE_PARTITION "Seçili bölümü silmek istediğinizden emin misiniz?"
+    IDS_WARN_DELETE_MBR_EXTENDED_PARTITION "Seçili genişletilmiş bölümü ve içerdiği TÜM mantıksal bölümleri silmek istediğinizden emin misiniz?"
 
-    IDS_ERROR_WRITE_PTABLE "Setup failed to write partition tables."
+    IDS_ERROR_WRITE_PTABLE "Kurulum bölüm tablolarını yazamadı."
 
-    IDS_ERROR_SYSTEM_PARTITION "The ReactOS Setup could not find a supported system partition\n\
-on your system or could not create a new one. Without such a partition\n\
-the Setup program cannot install ReactOS.\
-\nClick on OK to return to the partition selection list."
+    IDS_ERROR_SYSTEM_PARTITION "ReactOS Kurulumu sisteminizde desteklenen bir sistem bölümü bulamadı
+\n veya yeni bir bölüm oluşturamadı. Böyle bir bölüm olmadan
+\n Kurulum programı ReactOS'u yükleyemez.
+\n Bölüm seçim listesine dönmek için Tamam'a tıklayın."
 
-    IDS_ERROR_FORMAT_UNRECOGNIZED_VOLUME "Unrecognized volume while attempting to format the partition."
+    IDS_ERROR_FORMAT_UNRECOGNIZED_VOLUME "Bölüm biçimlendirilmeye çalışılırken tanınmayan birim."
 
-    IDS_ERROR_COULD_NOT_FORMAT "Setup is currently unable to format a partition in %s.\n\
-\nClick on OK to continue Setup.\
-\nClick on CANCEL to quit Setup."
+    IDS_ERROR_COULD_NOT_FORMAT "Kurulum şu anda bir bölümü şu şekilde biçimlendiremiyor: %s.\n\
+\n Kuruluma devam etmek için Tamam'a tıklayın.\
+\n Kurulumdan çıkmak için İPTAL üzerine tıklayın."
 
-    IDS_ERROR_FORMATTING_PARTITION "Setup is unable to format the partition:\n %s\n"
+    IDS_ERROR_FORMATTING_PARTITION "Kurulum bölümü biçimlendiremiyor:\n %s\n"
```

Comment:
@HBelusca was it an oopsie in en-US-template to have a space after the newline here, or was that intended for any reason?